AsyncDisplayKit/ASMultiplexImageNode.mm

```diff
@@ -63,7 +63,7 @@ @interface ASMultiplexImageNode ()
 
   // Image flags.
   BOOL _downloadsIntermediateImages; // Defaults to NO.
-  OSSpinLock _imageIdentifiersLock;
```

Comment:
lol - didn't realize this had been implemented with a Spin Lock.  Kill that beast!